.github/workflows/javadoc.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+      deployments: write
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: eu-west-2
+          role-to-assume: arn:aws:iam::${{ secrets.ABLY_AWS_ACCOUNT_ID_SDK }}:role/ably-sdk-builds-ably-java
+          role-session-name: "${{ github.run_id }}-${{ github.run_number }}"
+
+      - name: Set up the Java JDK
```

Comment:
The 'J' in JDK stands for 'Java'. But, arguably, the action name is self-explanatory so a `name` field for this step is superfluous. I would remove it.

Reply:
I agree. I will remove it.